SeleniumWebDriverBasics/SeleniumWebDriverBasics/Browser.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OpenQA.Selenium;
+
+namespace SeleniumWebDriverBasics
+{
+    public class Browser
+    {
+        public static IWebDriver driver;
```

Comment:
You can create property with public get and private set:
public static IWebDriver Driver { get; private set; }

This will simplify the code and allow you to delete the GetDriver() method because it is unnecessary.